### PR TITLE
chore: use ts

### LINKED ISSUE #21: Rewrite to TypeScript
title

### PR DESCRIPTION
**Pull request by plskz**
# Description

this PR closes #21

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,62 +1,26 @@
...
-  exec(`code ${path}`, (error, stdout, stderr) => {
-    if (error) return console.log(`error: ${error.message}`);
-    if (stderr) return console.log(`stderr: ${stderr}`);
-    // console.log(`stdout: ${stdout}`);
-    success(`\n\topening ${path}`);
-  });
-};
-
-// -------------------------------------------------------------
+console.log("test npm link");
```
Comment (ghost) [inline src/index.ts]:
```suggestion
```

File: src/index.ts [hunk 2]
```diff
@@ -142,3 +106,5 @@ export function run({ name, github, start, round = 1, vscode = true }) {
     console.error(err);
   }
 }
+
+console.log("finish runnin");
```
Comment (ghost) [inline src/index.ts]:
```suggestion
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (plskz) [reply src/index.ts]: oops I forgot to remove >_>

**Review by ghost — APPROVED** [review]
:tada:

**Pull request merged**